README.md

```diff
@@ -30,6 +30,20 @@ The flow is as follows:
 - [Deposit](https://etherspot.fyi/arka/api-calls/deposit) to the paymaster.
 - Include the paymaster data in the [UserOp](https://etherspot.fyi/account-abstraction/userops) to have the transaction sponsored.
 
+## 🐳 How to run (a Docker image)
+
+## Prerequisites
+* Docker Installed (https://www.docker.com/get-started/)
+* Edit docker-compose.yml for backend cron job and put a value in CRON_PRIVATE_KEY with the wallet private key you want to use to periodically update the paymaster contract price
```

Comment:
Pls have a default value setup. Which will run the docker with default configuration.

Reply:
all other env vars are of default values only the private_key is not there but its not mandatory and will skip if there is no private key given inside cron job